app/controllers/api/v1/screenshots_controller.rb

```diff
@@ -8,7 +8,17 @@ class ScreenshotsController < ApplicationController
 
       def index
         @screenshots = Screenshot.where(product_version_id: @version, locale_id: @locale)
-        render json: @screenshots, include: :images
+        @screenshot =  @screenshots.map { |screenshots| screenshots.as_json.merge(images:
+          screenshots.images.map { |image| url_for(image) }) }
+        if @screenshot.any?
+          render json: @screenshot
+        else
+          render json: {
+            status: 404,
+            error: :not_found,
+            message: 'Data not found'
```

Comment:
You would want the error message to give you an hint about what went wrong so that you can debug it. Include the `version` and `locale` here, so that you would an understanding of what went wrong to debug.

Reply:
Added locale and version in the error message. Thanks!